samples/release-management/csharp/ReleaseManagement/Models/ResourceModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="Resource.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace ReleaseManagement.Models
+{
+    using System.Collections.Generic;
+
+    public class ResourceModel
+    {
+        /// <summary>
+        /// Gets or sets project Id.
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets fields.
```

Comment:
which fields?

Reply:
Fields is the name of property coming in incoming payload